rskj-core/src/test/java/co/rsk/peg/FederationSupportTest.java

```diff
@@ -74,7 +74,7 @@ void whenNewFederationIsNullThenActiveFederationIsGenesisFederation() {
         when(provider.getNewFederation())
             .thenReturn(null);
         when(bridgeConstants.getGenesisFederation())
-            .thenReturn(genesisFederation);
+            .thenReturn((StandardMultisigFederation) genesisFederation);
```

Comment:
Shouldn't need to parse this

Reply:
it throws and error if not

Reply:
Shouldn't throw anymore, now that it was changed in BridgeConstants